iommi/query.py

```diff
@@ -606,12 +606,11 @@ def __init__(self, *, model=None, rows=None, filters=None, _filters_dict=None, a
         #   b) Not everything can be deepcopied.  And indeed in production a user
         #      hit a place where a not pickle-able object was contained in one of
         #      the copied places.
-        # So instead we do the below:
+        # So instead we do the below (the if just to avoid copying the namespace when we are
+        # not touching it).
         if 'form' in kwargs and 'fields' in kwargs['form'] and 'freetext' in kwargs['form']['fields']:
```

Comment:
I think the if could be removed. Having the copy done only sometimes does not help debugging.

Reply:
Also, I think you can drop the big comment.